pkg/apis/workflow/v1alpha1/http_types.go

```diff
@@ -12,6 +12,10 @@ type HTTPHeaderSource struct {
 
 type HTTPHeaders []HTTPHeader
 
+type HTTPBodySource struct {
+	Bytes []byte `json:"bytes,omitempty" protobuf:"bytes,1,opt,name=bytes"`
```

Comment:
Why are we using type `[]byte` here?

Reply:
I think this is probably correct - `string` would not work because it cannot encode certain characters, I think the JSON encoding is the body in base 64